### PR TITLE
Feat/k shareholder app first version

### PR DESCRIPTION
**Pull request by kimhansen**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [ ] If this change fixes a bug (or a performance problem), is a regression
  test (or a benchmark) included?
- [ ] Is a documentation update included (if this change modifies
  existing APIs, or introduces new ones)?

_NOTE: these things are not required to open a PR and can be done
afterwards / while the PR is open._

### CODE REVIEW

File: web/se/static/components/se/main.tag [hunk 1]
```diff
@@ -52,11 +50,16 @@
...
+        function shareholdersRouteSubId(level1, level2, level3) {
+            thizz.route_info.path = []
+            thizz.route_info.path.push('shareholders')
+            thizz.route_info.path.push(level1 || 'dashboard')
+            thizz.route_info.path.push(level2 || 'portfolio')
+            thizz.route_info.path.push(level3 || '')
 
-        function shareholdersRouteSubId(subsection, id) {
-            thizz.selected_id = id || ''
+            <!--  thizz.selected_id = id || ''
```
Comment (pedroarelo) [inline web/se/static/components/se/main.tag]: watch out with this comments, they are skreweing up the routing

File: web/se/static/components/se/shareholders/shareholders_dashboard.tag [hunk 2]
```diff
@@ -51,50 +51,101 @@
...
+            let profile = riot.app.store.getProfile();
+            let userEmail = profile.email || ""
+            thizz.portfolioList = []
+
+            thizz.totalShares = 0
+            thizz.amountInvested = 0
+            for (let i = 0; i < portfolioListData.shareholderOrganisations.length; i++) {
+                let organisation = portfolioListData.shareholderOrganisations[i]
+
+                <!--  let totalShares = 0
```
Comment (pedroarelo) [inline web/se/static/components/se/shareholders/shareholders_dashboard.tag]: watch this out again

File: web/se/static/components/se/main.tag (hunk 1 above)
Comment (kimhansen) [reply web/se/static/components/se/main.tag]: Oups. Thanks for catching this.

File: web/se/static/components/se/shareholders/shareholders_dashboard.tag (hunk 2 above)
Comment (kimhansen) [reply web/se/static/components/se/shareholders/shareholders_dashboard.tag]: Yes, I will go through the code to find any of those leftovers.

Comment (kimhansen) [reply web/se/static/components/se/shareholders/shareholders_dashboard.tag]: Fixed

**Comment by kimhansen** [comment]
I have done the requested changes. Please review again.

**Pull request merged**